packages/web-app-media-viewer/src/App.vue

```diff
@@ -8,17 +8,11 @@
     @keydown.esc="closeApp"
   >
     <h1 class="oc-invisible-sr" v-text="pageTitle" />
-    <transition
-      name="custom-classes-transition"
-      :enter-active-class="activeClass.enter"
-      :leave-active-class="activeClass.leave"
-    >
+    <transition name="custom-classes-transition">
```

Comment:
Please open an issue to add sliding animations to the mediaviewer again.

Reply:
https://github.com/owncloud/web/issues/6250